tool/test-annocheck.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh -eu
+# Run the `tool/test-annocheck.sh [binary files]` to check security issues.
+# E.g. `tool/test-annocheck.sh ruby libruby.so.3.2.0`.
+#
+# Use annocheck.
+# https://sourceware.org/annobin/
+
+set -x
+
+DOCKER="$(command -v docker || command -v podman)"
+TAG=ruby-fedora-annocheck
+TOOL_DIR=$(dirname "${0}")
+DOCKER_RUN_VOLUME_OPTS=
+
+if [ -z "${CI-}" ]; then
+  DOCKER_RUN_VOLUME_OPTS="-v $(pwd):/work"
+
+  # Build with volume option on local (non-CI).
+  "${DOCKER}" build --rm -t "${TAG}" ${TOOL_DIR}/annocheck/
+else
+  # TODO: Temporary workaround on CI to build by copying binary files from host
+  # to container without volume option, as I couldn't find a way to use volume
+  # in container in container on GitHub Actions <.github/workflows/compilers.yml>.
+  TAG="${TAG}-copy"
+  "${DOCKER}" build --rm -t "${TAG}" --build-arg=FILES="${*}" -f ${TOOL_DIR}/annocheck/Dockerfile-copy .
```

Comment:
On this line, I think `"${*}"` is right here rather than `"${@}"`. When I tested it by setting 2 binary files to `"${@}"`, the `docker build --build-arg` showed an error.